### PR TITLE
Changed busio.i2c to board.i2c

### PR DESCRIPTION
**Pull request by dherrada**
I'm not sure if I've got the hardware to test. I've got a gps featherwing, although I've never tried using it over i2c.

### CODE REVIEW

**Comment by dherrada** [comment]
In reference to #42

File: examples/gps_datalogging.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
 # uart = serial.Serial("/dev/ttyUSB0", baudrate=9600, timeout=10)
 
 # If using I2C, we'll create an I2C interface to talk to using default pins
-# i2c = busio.I2C(board.SCL, board.SDA)
+# i2c = board.I2C(board.SCL, board.SDA)
```
Comment (dhalbert) [inline examples/gps_datalogging.py]: these should all be `board.I2C()` without any arguments. `board.I2C()` uses the default pina.s

File: examples/gps_echotest.py [hunk 2]
```diff
@@ -18,7 +18,7 @@
 # uart = serial.Serial("/dev/ttyUSB0", baudrate=9600, timeout=10)
 
 # If using I2C, we'll create an I2C interface to talk to using default pins
-# i2c = busio.I2C(board.SCL, board.SDA)
+# i2c = board.I2C(board.SCL, board.SDA)
```
Comment (dhalbert) [inline examples/gps_echotest.py]: ditto

File: examples/gps_simpletest.py [hunk 3]
```diff
@@ -18,7 +18,7 @@
 # uart = serial.Serial("/dev/ttyUSB0", baudrate=9600, timeout=10)
 
 # If using I2C, we'll create an I2C interface to talk to using default pins
-# i2c = busio.I2C(board.SCL, board.SDA)
+# i2c = board.I2C(board.SCL, board.SDA)
```
Comment (dhalbert) [inline examples/gps_simpletest.py]: ditto

File: examples/gps_datalogging.py (hunk 1 above)
Comment (dherrada) [reply examples/gps_datalogging.py]: Ah, ok. I'll fix that when I've got a sec

**Pull request merged**